src/template-tags/tickets.php

```diff
@@ -419,26 +419,25 @@ function tribe_tickets_is_current_time_in_date_window( $post_id ) {
 		$tickets               = Tribe__Tickets__Tickets::get_all_event_tickets( $post_id );
 		$default_provider      = Tribe__Tickets__Tickets::get_event_ticket_provider( $post_id );
 
-		/** @var Tribe__Tickets__Ticket_Object $ticket */
-		foreach ( $tickets as $ticket ) {
-			$ticket_provider = $ticket->get_provider();
+		if ( false !== $default_provider ) {
```

Comment:
Added the outer conditional because it was referencing `$default_provider` multiple times as a condition that skips all the logic within the foreach